components/CovidMap.js

```diff
@@ -0,0 +1,702 @@
+  height: '35', 
+  className: 'custom-clustericon-first'
+}
+
+const clusterStyleFirstLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-first'
+}
+
+const clusterStyleSecondSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleSecondMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleSecondLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleThirdSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-third'
+}
+
+const clusterStyleThirdMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-third'
+}
+
+const clusterStyleThirdLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-third'
+}
+
+const formStyle = {
+  width: '50%',
+  float: 'left', 
+}
+
+const legendStyle = {
+  width: '50%', 
+  float: 'right',
+}
+
+const legendItemStyle = {
+  width: '100%',
+  marginBottom: '5px',
+}
+
+const legendTextStyle = {
+  width: '90%',
+  height: '20px',
+  float: 'left',
+  textAlign: 'right',
+}
+
+const legendIconStyle = {
+  minWidth: '30px',
+  height: '20px',
+  float: 'right',
+  display: 'inline-block',
+}
+
+const legendIconFirst = {
+  background: '#0B5D93',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'right',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+const legendIconSecond = {
+  background: '#634562',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'left',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+const legendIconThird = {
+  backgroundColor: '#CE2727',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'left',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+
+class Map extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      map: null,
+      firstClusterer: null,
+      secondClusterer: null, 
+      thirdClusterer: null,
+      fetchedMap: false,
+      data: [],
+      selectUpdate: true,
+      selectedOption: props.value || 'all',
+      firstLegendText: 'All Deaths',
+      secondLegendText: '',
+      thirdLegendText: ''
+    }
+  }
+
+  getGoogleMaps() {
+    // If we haven't already defined the promise, define it
+    if (!this.googleMapsPromise) {
+      this.googleMapsPromise = new Promise((resolve) => {
+        // Add a global handler for when the API finishes loading
+        window.resolveGoogleMapsPromise = () => {
+          // Resolve the promise
+          const google = window.google;
+          resolve(google);
+
+          // Tidy up
+          delete window.resolveGoogleMapsPromise;
+        };
+
+        // Load the Google Maps API
+        const API = 'AIzaSyDAh7M89BnID8kGVXBrNtxJfD-jjDDFRCg';
```

Comment:
We can add the API key into our Netlify variables so that it doesn't need to be stored in Github. Variables can be pulled in via [Netlify functions](https://functions.netlify.com/). 

We've done this for our PayPal API keys. You can see the fetch [here](https://github.com/texas-justice-initiative/website-nextjs/blob/master/components/forms/ReviewForm.js), and the Netlify function we created [here](https://github.com/texas-justice-initiative/website-nextjs/blob/master/functions/paypal_params.js).

Reply:
Ok, I think I see how it works, I'll give it a try locally and see how it goes.